c7n/resources/dynamodb.py

```diff
@@ -687,28 +687,23 @@ class TableConsecutiveBackups(Filter):
                   - type: consecutive-backups
                     count: 7
                     period: days
-                    backuptype: SYSTEM
-                    status: AVAILABLE
+                    status: 'COMPLETED'
     """
     schema = type_schema('consecutive-backups', count={'type': 'number', 'minimum': 1},
         period={'enum': ['hours', 'days', 'weeks']},
-        backuptype={'enum': ['SYSTEM', 'USER', 'AWS_BACKUP', 'ALL']},
-        status={'enum': ['AVAILABLE', 'CREATING', 'DELETED']},
-        required=['count', 'period', 'status', 'backuptype'])
+        status={'enum': ['COMPLETED', 'PARTIAL', 'DELETING', 'EXPIRED']},
+        required=['count', 'period', 'status'])
     permissions = ('dynamodb:ListBackups', 'dynamodb:DescribeBackup', 'dynamodb:DescribeTable', )
     annotation = 'c7n:DynamodbBackups'
 
-    def process_resource_set(self, client, resources, lbdate):
-        paginator = client.get_paginator('list_backups')
+    def process_resource_set(self, resources):
+        client = local_session(self.manager.session_factory).client('backup')
+        paginator = client.get_paginator('list_recovery_points_by_resource')
```

Comment:
if we want to filter on aws backups this should be a separate generic filter, as this not only breaks the policy schema but also the inherent behavior as well. users that have backups created natively in dynamodb would not see their backups reflected in this API call

Reply:
@thisisshi i am not sure how this can be done using a generic filter ... the "list_recovery_points_by_resource" is a related resource call and is just used to collect the backup related info for the resource (table) and append to resource metadata.  the original call used , doesn't return the backup metadata for the table that were created using the aws backup service. Only the 'list_recovery_points_by_resource' call returns all the type of backups but doesn't differentiate on the backup type, so the schema will have to be modified anyways.

Reply:
what i'm saying is that this will need to be a separate filter and one that can be generic and not confined to the dynamodb resource. Creating breaking changes on the extant filter is not appropriate in addition, what you're describing here with the `list_recovery_points_by_resource` can be generically applied to any resource that supports aws backup